readme.md

```diff
@@ -72,6 +98,59 @@ Temporal is a [durable execution system](https://temporal.io/blog/building-relia
 
 - [iWF](https://github.com/indeedeng/iwf) - DSL workflow framework.
 
+## Videos
+
+### Why Temporal
+
+- [Replay Keynote](https://www.youtube.com/watch?v=W0Ygep6iCJY) - Durable execution, Nexus, and Cloud.
+- [Glovo: Fault tolerant distributed microservices with Temporal](https://youtu.be/6lSuDRRFgyY)
+- [TheDevConf: Fault Tolerant, Distributed Microservices Orchestration with Temporal](https://www.youtube.com/watch?v=6T6zVZHU7_Q) - Introduction to Temporal ([slides](https://temporal-intro-and-demo.netlify.app/)).
```

Comment:
should an intro be down in the "using temporal" section? Or maybe we change the "explanation" here to give a little more of a hint as to how this video answers the "why temporal" question